docs/v0.7/en/reference/sql/information-schema/columns.md

```diff
@@ -0,0 +1,103 @@
+| table_catalog  | String |      | NO   |         | FIELD         |
+| table_schema   | String |      | NO   |         | FIELD         |
+| table_name     | String |      | NO   |         | FIELD         |
+| column_name    | String |      | NO   |         | FIELD         |
+| data_type      | String |      | NO   |         | FIELD         |
+| semantic_type  | String |      | NO   |         | FIELD         |
+| column_default | String |      | YES  |         | FIELD         |
+| is_nullable    | String |      | NO   |         | FIELD         |
+| column_type    | String |      | NO   |         | FIELD         |
+| column_comment | String |      | YES  |         | FIELD         |
++----------------+--------+------+------+---------+---------------+
+```
+Create a table `public.t1` and query the information in the `COLUMNS` table:
+
+```sql
+CREATE TABLE public.t1 (h STRING, v FLOAT64, ts TIMESTAMP DEFAULT CURRENT_TIMESTAMP() TIME INDEX, PRIMARY KEY(h));
+SELECT * FROM COLUMNS WHERE table_schema='public' AND TABLE_NAME='t1'\G
+```
+
+The output is as follows:
+
+```sql
+*************************** 1. row ***************************
+ table_catalog: greptime
+  table_schema: public
+    table_name: t1
+   column_name: h
+     data_type: String
+ semantic_type: TAG
+column_default: NULL
+   is_nullable: Yes
+   column_type: String
+column_comment: NULL
+*************************** 2. row ***************************
+ table_catalog: greptime
+  table_schema: public
+    table_name: t1
+   column_name: v
+     data_type: Float64
+ semantic_type: FIELD
+column_default: NULL
+   is_nullable: Yes
+   column_type: Float64
+column_comment: NULL
+*************************** 3. row ***************************
+ table_catalog: greptime
+  table_schema: public
+    table_name: t1
+   column_name: ts
+     data_type: TimestampMillisecond
+ semantic_type: TIMESTAMP
+column_default: current_timestamp()
+   is_nullable: No
+   column_type: TimestampMillisecond
+column_comment: NULL
+3 rows in set (0.01 sec)
+```
+
+The description of columns in the `COLUMNS` table is as follows:
+
+- `table_catalog`: The name of the catalog to which the table with the column belongs. The value is always `greptime`.
+- `table_schema`: The name of the schema in which the table with the column is located.
+- `table_name`: The name of the table with the column.
+- `column_name`: The name of the column.
+- `data_type`: The type of data in the column.
+- `semantic_type`: The type of the column. It can be one of the following: `TAG`, `FIELD`, or `TIMESTAMP`.
+- `column_default`: The default value of the column. If the explicit default value is `NULL`, or if the column definition does not include the `default` clause, this value is `NULL`.
+- `is_nullable`: Whether the column is nullable. If the column can store null values, this value is `YES`; otherwise, it is `NO`.
+- `column_type`: The data type of the column. It is the same as the `DATA_TYPE` column.
```

Comment:
@killme2008 please confirm whether they are the same

Reply:
If I wrote it, it must be the same. :D